drivers/SmartThings/philips-hue/src/handlers.lua

```diff
@@ -250,7 +254,39 @@ end
 ---@param bridge_device HueBridgeDevice
 local function do_refresh_all_for_bridge(driver, bridge_device)
   local child_devices = bridge_device:get_child_list() --[=[@as HueChildDevice[]]=]
+  local num_devices_expected = 0
+  for _, device in ipairs(driver:get_devices()) do
+    if not device.parent_device_id then
+      log.warn_with(string.format("Device %s does not have a parent_device_id",
+        (device.label or device.id or "unknown device")))
+    elseif device.parent_device_id == bridge_device.id then
+      num_devices_expected = num_devices_expected + 1
+    end
+  end
+  log.trace_with({ hub_logs = true },
```

Comment:
Info level here too? Is this info that you'd want to confirm in most investigations?